.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,15 @@
+name: Feature request 💄
+description: Suggest an improvement
+labels: ["feature request"]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for contributing by creating an issue! ❤️
+  - type: textarea
+    attributes:
+      label: Summary 💡
+      description: Describe how it should work.
+  - type: textarea
+    attributes:
+      label: Motivation 🔦
+      description: What are you trying to accomplish? How has the lack of this feature affected you? Providing context helps us come up with a solution that is more useful in the real world.
```

```suggestion
      description: >
        What are you trying to accomplish? How has the lack of this feature affected you?
        Providing context helps us come up with a solution that is more useful in the real world.
```
Just for the sake of readability, maybe use [YAML's `>`](https://yaml.org/spec/1.2.2/#folded-style) to wrap the lines?